components/imageCard.js

```diff
@@ -19,27 +27,27 @@ export default function ImageCard() {
         setDisplayImage(randomizeImage(serverData.items));
       });
   }, []);
```

Comment:
you need a `.catch((err)=>{})` block here to handle errors (e.g. wrong url, server down, no internet etc.)

Reply:
I kind of rebuild the whole code and it surprisingly still works.